README.md

```diff
@@ -1,2 +1,397 @@
+
+## Prerequisites
+
+- [Docker and docker-compose](https://docs.docker.com/get-docker/)
+- Pull the docker images according to the docker-compose file or run `docker-compose pull`
+
+## Run Your Tests
+
+In this part we will run the tests in current folder in different confirgurations on docker
+
+### Run Single Test on Docker
+
+In this section we will run a test on a browser on a selenium grid hosted in docker
+
+- How to run the test?
+
+    - Start the Selenium Grid first by running the following command:
+
+  ```sh
+  docker-compose up -d
+  ```
+
+    - To run the test go to your terminal or command line and paste the following command:
+
+  ```sh
+  mvn test -P docker -Dtest-name="test name"
+  ```
+
+    - After tests are complete stop the Selenium grid by running the following command:
+      
+  ```sh
+  docker-compose down
+  ```
+
+- Output
+
+  It will not open a browser on your machine but run the test.
+
+    - Console output
+
+  ```sh
+  ~paste console output for single test!~
+  ```
+
+    - Browser will not be launched here as we are running inside Docker environment.
+
+### Run Parallel Test on Docker
+
+In this section we will run some tests on a browser on a selenium grid hosted in docker in parallel across multiple browsers
+
+- How to run the test?
+
+    - Start the Selenium Grid first by running the following command:
+
+  ```sh
+  docker-compose up -d
+  docker-compose scale chromenode=3
+  ```
+
+    - To run the test go to your terminal or command line and paste the following command:
+
+  ```sh
+  mvn test -P docker-parallel
+  ```
+
+    - After the tests are complete stop the Selenium grid by running the following command:
+
+  ```sh
+  docker-copmpose down
+  ```
+
+- Output
+
+  It will not open a browser on your machine but run the test.
+
+    - Console output
+
+  ```sh
+  ~paste console output for parallel test!~
+  ```
+
+    - Browser will not be launched here as we are running inside Docker environment.
+
+---
+
+# BrowserStack
+
+App & Browser Testing Made Easy!! Give your users a seamless experience by testing on 2000+ real devices and browsers. Don't compromise with emulators and simulators.
+
+## Prerequisites
+
+- Create an account on BrowserStack if not already
+- Export the environment variables for the Username and Access Key of your BrowserStack account.
+
+    - For \*nix based and Mac machines:
+
+  ```sh
+  export BROWSERSTACK_USERNAME=<browserstack-username> &&
+  export BROWSERSTACK_ACCESS_KEY=<browserstack-access-key>
+  ```
+
+    - For Windows:
+
+  ```shell
+  set BROWSERSTACK_USERNAME=<browserstack-username>
+  set BROWSERSTACK_ACCESS_KEY=<browserstack-access-key>
+  ```
+
+- Clone the browserstack-canonical-app repository using `git clone https://github.com/browserstack/browserstack-demo-app`
```

Comment:
this should be in the bstack-local testing session. Also, mention that users should follow the README.md of the app repo to start the local server as a Pre-requisite in the Local section.